broker-core/src/main/java/org/apache/qpid/server/store/JsonFileConfigStore.java

```diff
@@ -156,49 +156,52 @@ protected boolean load(final ConfiguredObjectRecord... initialRecords)
 
             boolean updated = false;
             Collection<ConfiguredObjectRecord> records = Collections.emptyList();
-            ConfiguredObjectRecordConverter configuredObjectRecordConverter =
+            final ConfiguredObjectRecordConverter configuredObjectRecordConverter =
                     new ConfiguredObjectRecordConverter(_parent.getModel());
 
-            records = configuredObjectRecordConverter.readFromJson(_rootClass, _parent, new FileReader(configFile));
-
-            if(_rootClass == null)
+            try (FileReader configFileReader =  new FileReader(configFile))
```

Comment:
This change is unnecessary.  The `configFileReader` is closed in `ObjectMapper`. If you really need to make sure that `configFileReader` is closed in the code where it is created, you can simply add `try-with-resource` block around  the code reading from json
```
try (FileReader configFileReader =  new FileReader(configFile))
{
    records = configuredObjectRecordConverter.readFromJson(_rootClass, _parent, configFileReader);
}
```